File: ProBen.Domain/Servidor.cs
using System;
using System.Collections.Generic;

namespace ProBen.Domain
{
    public class Servidor
    {
        public int Id { get; set; }
        public string Nome { get; set; }
        public string Cpf { get; set; }
        public int Matricula { get; set; }
        public DateTime DataCriacao { get; set; }
        public string Telefone { get; set; }
        public string Email { get; set; }
        public int BeneficioId { get; set; }
        public Beneficio Beneficio { get; }
        public List<Orgao> Orgaos { get; set; }
    }
}

File: ProBen-App/src/app/_models/Servidor.ts
import { Orgao } from "./Orgao";

export interface Servidor {
    id: number;
    nome: string;
    cpf: string;
    matricula: number;
    dataCriacao: Date;
    telefone: string;
    email: string;
    beneficioId: number;
    orgaos: Orgao[];
}
